scripts/autolabels.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+ : <<'BASH_MODULE_DOCS'
+
+This file contains the dictionary for converting a modified dir into a github label,
+as well as the script to perform the conversion.
+
+Rules:
+* only lines starting with `|t-` are considered;
+* repeated spaces, commas and pipes (|) are ignored;
+* `Mathlib` is added by default to each `Root folder`.
+
+Exception: `Tactic/Linter` is converted into `Tactic-Linter` internally, to simplify separating it from
+`Tactic/anything_other_than_linter`.
+
+|Label                   |Root folders                                                                       |
+|-                       |-                                                                                  |
+|t-algebra               | Algebra, FieldTheory, RingTheory, GroupTheory, RepresentationTheory, LinearAlgebra|
+|t-algebraic-geometry    | AlgebraicGeometry, Geometry/RingedSpace                                           |
+|t-euclidean-geometry    | Geometry.Euclidean                                                                |
+|t-differential-geometry | Geometry/Manifold                                                                 |
+|t-analysis              | Analysis                                                                          |
+|t-category-theory       | CategoryTheory                                                                    |
+|t-combinatorics         | Combinatorics                                                                     |
+|t-computability         | Computability                                                                     |
+|t-condensed             | Condensed                                                                         |
+|t-data                  | Data                                                                              |
+|t-dynamics              | Dynamics                                                                          |
+|t-linter                | Tactic/Linter                                                                     |
+|t-logic                 | Logic, ModelTheory                                                                |
+|t-measure-probability   | MeasureTheory, Probability, InformationTheory                                     |
+|t-meta                  | Tactic, Lean, Init, Util, Mathport, Control, Testing                              |
```

Comment:
I would rather *not* apply `Init` here: that directory is a random mix of things.

Reply:
Sorry for bringing this up now...

Reply:
Removed it!  I should have removed it to begin with, but I was more focused on getting the script work, than actually looking at the labeling.